VSRAD.Package/DebugVisualizer/SliceVisualizer/SliceVisualizerContext.cs

```diff
@@ -48,6 +48,48 @@ public SliceVisualizerContext(Options.ProjectOptions options, VisualizerContext
             _windowVisibilityEvents.WindowHiding += OnToolWindowVisibilityChanged;
         }
 
+        public void SetStatusString(int row, int column, string val)
+        {
+            if (row < 0 || column < 0 || string.IsNullOrEmpty(val))
+            {
+                StatusString = "";
+                return;
+            }
+
+            if (SelectedWatchView.IsInactiveCell(row, column))
+            {
+                StatusString = "Out of bounds";
+                return;
+            }
+
+            var gNum = SelectedWatchView.GetGroupIndex(row, column);
+            var lNum = SelectedWatchView.GetLaneIndex(column);
+            StatusString = $"Group# {gNum}, Column# {lNum}, Value: {val}";
+        }
+
+        public void NavigateToCell(int sliceRowIndex, int sliceColumnIndex)
+        {
+            Options.VisualizerOptions.NDRange3D = false; // disable NDRange3D since we dont have implementation of navigation for it
```

Comment:
See my suggestion about replacing `SetGroupIndex` with `GoToGroup`, which doesn't require this hack.